kernels/volk/volk_32fc_index_max_32u.h

```diff
@@ -385,12 +313,12 @@ static inline void
 #define INCLUDED_volk_32fc_index_max_32u_u_H
```

Comment:
I suggest to simplify this include guard. i.e. only one include guard for all kernel versions.